doc/articles/features/windows-system-power.md

```diff
@@ -15,5 +15,17 @@ uid: Uno.Features.WSPower
 **iOS**
 - `RemainingDischargeTime` and `RemainingDischargeTimeChanged` events not supported (platform does not offer appropriate APIs)
 
+**WASM**
+
+Before any of the properties/events of `PowerManager` can be accessed, the class needs to be initialized via a platform-specific `InitializeAsync` method:
+
+```csharp
+#if __WASM__
+var isAvailable = await PowerManager.InitializeAsync();
+#endif
```

Comment:
It's unclear where does the user needs to be put this code. In the class library, `#if __WASM__` wouldn't work.

Reply:
We probably should make this API available in all platforms for consistency, and let the user access it without the need for any `#if` directives.

Reply:
All platforms except Windows I guess 🤔 . `#if HAS_UNO` will still be needed...

Reply:
Which actually makes me think - maybe we should include `__CROSSRUNTIME__` symbol in Uno apps as well (to be able to `#if` WASM and Skia)